docs/conf.py

```diff
@@ -21,13 +21,16 @@
 from pathlib import Path
 
 here = Path(__file__).absolute().parent
-version = (here.parent / 'VERSION').read_text().strip()
+
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('..'))
 
+from juju.version import CLIENT_VERSION
```

Comment:
Imports should be at the top no?

Reply:
This one should stay there because it won't work before the `sys.path.insert(0, os.path.abspath('..'))` line above